domain/src/main/java/org/oppia/android/domain/topic/TopicListController.kt

```diff
@@ -456,25 +429,33 @@ class TopicListController @Inject constructor(
     )
   }
 
+  private fun getDependentLisOfTopicForOngoingTopic(topicId: String): List<String> {
+    val listOfTopicIds = mutableListOf<String>()
```

Comment:
Please add a TODO here on https://github.com/oppia/oppia-android/issues/2550 to update this to not be hardcoded (just so that we track that long-term; it doesn't need to be resolved as part of this project).

Reply:
Done.